graphene/core/classtypes/uniontype.py

```diff
@@ -35,6 +35,6 @@ def internal_type(cls, schema):
         return GraphQLUnionType(
             cls._meta.type_name,
             types=list(map(schema.T, cls._meta.types)),
-            resolve_type=cls._resolve_type,
+            resolve_type=lambda instance, info: cls._resolve_type(schema, instance, info),
```

Comment:
For being in constance with the [implementation of `Interface`](https://github.com/graphql-python/graphene/blob/master/graphene/core/classtypes/interface.py#L51), maybe the following will be better.

```python
from functools import partial
...
            resolve_type=partial(cls._resolve_type, schema),
```